doc/make.bat

```diff
@@ -41,6 +44,9 @@ if "%1" == "clean" (
 )
 
 if "%1" == "html" (
+	mkdir source/api
+	%PYTHON% tools/build_modref_templates.py 
+	cd source && %PYTHON% random_gallery.py
```

Comment:
In the Makefile we also build the API for the `latex` target. Might be more elegant to have a separate conditional for this.